### PR TITLE
upgraded tar to 6.2.1

### PR DESCRIPTION
**Pull request by cs-raj**
@oclif/plugin-plugins: 4.1.9 -> 5.0.0
tar: 6.1.3 -> 6.2.1

### CODE REVIEW

**Review by aman19K — APPROVED** [review]

File: packages/contentstack-audit/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@contentstack/cli-command": "~1.2.16",
     "@contentstack/cli-utilities": "~1.6.0",
     "@oclif/plugin-help": "^5",
-    "@oclif/plugin-plugins": "^4.1.9",
+    "@oclif/plugin-plugins": "^5.0.0",
```
Comment (antonyagustine) [inline packages/contentstack-audit/package.json:24]: @cs-raj have you validated if there are any breaking changes with this MAJOR version? As I can see, they released so many prereleases for `v5-x.x-beta` to test, I guess.

File: packages/contentstack-import/package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "@types/mocha": "^8.2.2",
     "@types/node": "^14.14.32",
     "@types/sinon": "^10.0.2",
-    "@types/tar": "^4.0.3",
+    "@types/tar": "^6.1.3",
```
Comment (antonyagustine) [inline packages/contentstack-import/package.json:37]: Is there any breaking changes for this package?

**Review by antonyagustine — COMMENTED** [review]
It could be better to mention in the description about the change if we update any package to MAJOR version.

File: packages/contentstack-audit/package.json (hunk 1 above)
Comment (cs-raj) [reply packages/contentstack-audit/package.json:24]: I didn't find any breaking changes

File: packages/contentstack-import/package.json (hunk 2 above)
Comment (cs-raj) [reply packages/contentstack-import/package.json:37]: No

**Review by antonyagustine — APPROVED** [review]

**Pull request merged**